test/testenv/util.go

```diff
@@ -464,21 +464,26 @@ func newStandaloneWithSpec(name, ns string, spec enterprisev1.StandaloneSpec) *e
 	return &new
 }
 
-// DumpGetPods prints list of pods in the namespace
-func DumpGetPods(ns string) {
+// DumpGetPods prints and returns list of pods in the namespace
+func DumpGetPods(ns string) []string {
 	output, err := exec.Command("kubectl", "get", "pods", "-n", ns).Output()
+	var splunkPods []string
 	if err != nil {
 		cmd := fmt.Sprintf("kubectl get pods -n %s", ns)
 		logf.Log.Error(err, "Failed to execute command", "command", cmd)
-	} else {
-		for _, line := range strings.Split(string(output), "\n") {
-			logf.Log.Info(line)
+		return nil
+	}
+	for _, line := range strings.Split(string(output), "\n") {
+		logf.Log.Info(line)
```

Comment:
Nit: Is this log needed for production?

Reply:
We use this in every verification to print list of PODS in a namespace. It is very useful for debug.